account_invoice_under_payment/models/account_move_line.py

```diff
@@ -13,7 +13,7 @@ class AccountMoveline(models.Model):
     @api.depends('payment_line_ids.failed', 'payment_line_ids.move_line_id')
     def _check_payment(self):
         for record in self:
-            if any([not x.failed for x in record.payment_line_ids]):
+            if any([x.failed for x in record.payment_line_ids]):
```

Comment:
We tested and it worked only when the 'not' was removed. would you please test and revert?